src/hammer-vlsi/pcb/generic/__init__.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+#  hammer-vlsi plugin for Cadence Innovus.
```

Comment:
copypaste?

Reply:
Oops, fixed. Also added some text about what deliverables it builds.